src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
Please use * selector only for box-sizing rule,  move font-family and margin: 0 to body selector